rfcs/0173-hotline.md

```diff
@@ -0,0 +1,234 @@
+# RFC nixos hotline
+
+---
+feature: NixOS PR hotline
+start-date: 2024-04-01
+author: trollulus
+co-authors: riotbib, rtrollreal, b-kenji, zmberber, kmein
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Bring the Power of German bureaucracy to Nixpkgs by using a sophisticated telephone system to interact with pull requests.
+
+# Motivation
+[motivation]: #motivation
+
+As recent surveys <sup>[citation needed]</sup> have shown a near 80% dominance of the German language in nixpkgs, we want to align the nixpkgs workflow with the natural way German people interact with complex systems.
+Also, there are a lot of open pull requests and the amount of work doesn't seem to scale for our small group of contributors. By establishing a way to interact with nixpkgs without the need for a computer or a smartphone, we unlock a lot of potential contributors.
+
+# Detailed design
+[design]: #detailed-design
+
+We establish a hotline ``+49 0800 64967(NIXOS)`` where users can interact with nixpkgs. The system will be in German since that is the most spoken language in the nixpkgs community.
```

Comment:
I believe this rfc is missing the phone service provider we will register the number at. As a native potato person I would recommend the Deutsche Post Ag

Reply:
I think 0800 can be registered for free at the BNetzA https://www.bundesnetzagentur.de/DE/Fachthemen/Telekommunikation/Nummerierung/0800/0800_node.html

Reply:
I think it would have to be `+49 0800 649 7546` (NIXPKGS)`, since vanity numberes apparently require seven numbers.

![image](https://github.com/NixOS/rfcs/assets/131599/d907572e-b949-429d-806a-faaecce49749)

https://www.bundesnetzagentur.de/SharedDocs/Downloads/DE/Sachgebiete/Telekommunikation/Unternehmen_Institutionen/Nummerierung/Rufnummern/0800/0800_Antragsverfahren.pdf?__blob=publicationFile&v=1

Reply:
![grafik](https://github.com/NixOS/rfcs/assets/36541313/a6363c83-ab94-485b-8bbf-fcc8a3ef1988)
That seems to be the case, I've checked and `+49 0800 649 7546` (NIXPKGS) is free.

Reply:
Just briefly picking this up, the last number should be 7, not 6; thus: `+49 0800 6497547`.

Happily, my registration process for the registration was successfull:

>  Ihr Antrag (0800) ist fertiggestellt und hat die Antragsnummer [REDACTED].

I'll get back to you, once I received the snail mail.